travvy/src/components/Onboarding.vue

```diff
@@ -4,19 +4,27 @@
     
     <!-- Onboarding fields that the user must fill in in order to get personalized recommendations --> 
     <h1>🎉 Welcome to Travvy, we'd love to learn more about you! 🎉</h1>
+    
+  <form>
     <h3>Please fill in the following information in order to get a custom travel experience.</h3>
     <br>
+    <p v-if="errors.length">
+    <b class="red">Please correct the following error(s):</b>
+    <!-- this will bind all the errors together after the form is completed and return them to the user-->
+      <u1 class="red">
+        <li v-bind="error in errors"> {{ errors }}</li>
+      </u1>
+    </p>
```

Comment:
Perfect, thanks so much for adding this error handling in, this will definitely help users identify whether their input follows the correct format going forward. Out of curiosity, this would just return a list of errors of places the user inputted incorrect format/info at the top of the page, correct?

Reply:
Correct, I implemented similar functionality as the SignUp page's error handling in order to keep consistent UI and allow the user to enjoy a positive user experience